Formula/age.rb

```diff
@@ -0,0 +1,22 @@
+class Age < Formula
+  desc "Simple, modern, secure file encryption"
+  homepage "https://filippo.io/age"
+  url "https://github.com/FiloSottile/age/archive/v1.0.0.tar.gz"
+  sha256 "8d27684f62f9dc74014035e31619e2e07f8b56257b1075560456cbf05ddbcfce"
+  license "BSD-3-Clause"
+  head "https://github.com/FiloSottile/age.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-X main.Version=v#{version}"), "filippo.io/age/cmd/..."
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-X main.Version=v#{version}"), "-o", bin, "filippo.io/age/cmd/..."
```
Oops, I didn't notice that `std_go_args` was not equivalent to the original formulation. Mea culpa. I think this should work.